client/my-sites/plan-features-2023-grid/index.tsx

```diff
@@ -112,6 +120,7 @@ export type PlanFeatures2023GridProps = {
 	intent?: PlansIntent;
 	isGlobalStylesOnPersonal?: boolean;
 	showLegacyStorageFeature?: boolean;
+	showUpgradeableStorage: boolean;
```

Comment:
let's add a brief comment above/next to this to clarify what it is for and that it's the result of a feature flag - pls :-)

Reply:
Resolved in [500cc89](https://github.com/Automattic/wp-calypso/pull/79041/commits/500cc896001ca24578a292505c3dd710b619e99f)